frame/treasury/src/mock.rs

```diff
@@ -116,12 +116,12 @@ pub fn new_test_ext() -> sp_io::TestExternalities {
 	}
 	.assimilate_storage(&mut t)
 	.unwrap();
-	pallet_kton::GenesisConfig::<Test> {
-		// Total issuance will be 200 with treasury account initialized at ED.
-		balances: vec![(0, 100), (1, 98), (2, 1)],
-	}
-	.assimilate_storage(&mut t)
-	.unwrap();
+	// pallet_kton::GenesisConfig::<Test> {
+	// 	// Total issuance will be 200 with treasury account initialized at ED.
+	// 	balances: vec![(0, 100), (1, 98), (2, 1)],
+	// }
+	// .assimilate_storage(&mut t)
+	// .unwrap();
```

Comment:
KTON's GenesisConfig will override the RING's, haven't figured out why yet.

Reply:
I'll make a pr to fix this